install-unix.sh

```diff
@@ -42,11 +42,11 @@ fi
 echo installling event_name=$1
 
 vm_cmd="cd ~/uber/ && ./run-simple-deploy.sh $1 && rm -f ./run-simple-deploy.sh"
-git clone https://github.com/magfest/ubersystem-deploy
+test -d ubersystem-deploy || git clone https://github.com/magfest/ubersystem-deploy
 cd ubersystem-deploy
 vagrant up
 cp ../installfiles/run-simple-deploy.sh .
-vagrant ssh -c \'$vm_cmd\'
+vagrant ssh -c "$vm_cmd"
 
```

Comment:
ah, this makes a ton of sense, it was actually separating out $vm_cmd recently that made it lose the correct quotes, this fix is great, thanks.  I don't know how it was working correctly on my local box (Cygwin on windows) actually... weird. +1 on this